portal-frontend/src/app/features/public/application/submission/naru-details/naru-details.component.scss

```diff
@@ -1,9 +1,7 @@
-@use '../../../../../../styles/functions' as *;
-
 .soil-table {
   display: grid;
   grid-template-columns: max-content max-content;
```

Comment:
Missing responsive view (I also realized ALCS and Portal Applicant View missed NARU, I will update it after this PR is merged)